js/scripts.js

```diff
@@ -0,0 +1,46 @@
+// Jacob here, found something which might work.
+// In the html, give the section/header/div the following id to show the clock:: 'clock'
+// For example: <h1 class="text-center" id="clock"></h1>
+
+// MAKE THE ARRAYS HERE.
+// (will need at least 2: one for the titles and one for the actual description)
+
+// PUT THE boolean VALUE HERE (for preventing the users from spamming the "refresh" feature)
+var hammertime = true;
+
+// PUT THE ARRAY REFRESHER HERE
+// (need to dig up the randomizer selector from previous class or find one with internet)
+function array_refresh (){
+  console.log("Wait, is this midnight or early morning?"); //just for checking that this works as intended with displayClock func.
+};
+
+
+//HERE BEGINS THE CODE FOR THE COUNTDOWN CLOCK ---------------------------------
+function checkTime (i) {
```

Comment:
Below you seem to pass things like hours, minutes, and seconds. A variable name like "time" or "timeSegments" or something might be more clear.